src/bindings/python/tests/test_runtime/test_core.py

```diff
@@ -323,6 +323,8 @@ def test_unregister_plugin(device):
 
 
 @pytest.mark.template_plugin()
+@pytest.mark.skip(reason="Sporadically failed on mac with error:  Cannot add extension."
```

Comment:
It looks to be the only test that covers loading of so extension library in Python, and it is disabled. @jiwaszki, @akuporos, please comment on that. We have a similar issue: https://github.com/openvinotoolkit/openvino/issues/18834.

Reply:
The corresponding ticket in JIRA: CVS-95983
Actually the reason of disablement was that it sometimes failed to find extension on Mac OS.